src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1474,6 +1476,47 @@ protected override void InvokeDefaultAction(string path)
             return true;
         }
 
+        //Returns true if a given DirectoryInfo path is a recursive sym link.
+        private bool IsRecursiveSymLink(DirectoryInfo path)
+        {
+            string readSymLink = "";
+
+            if (Platform.IsWindows)
+            {
+                PSObject obj = PSObject.AsPSObject(path);
+
+                try
+                {
+                    readSymLink = InternalSymbolicLinkLinkCodeMethods.GetTarget(obj).First().ToString();
```

Comment:
>GetTarget(obj).First().ToString();

`InternalSymbolicLinkLinkCodeMethods.GetTarget` returns `IEnumerable<string>`, so the `ToString()` is unnecessary.